### PR TITLE
Fix/NSA-158/collapse a long outcome declarations names

### PR DESCRIPTION
**Pull request by andrey-ponamarev**
**Related to task:**  https://oat-sa.atlassian.net/browse/NSA-158
**Description:** Collapse a long outcome declarations names
Before | After
------------ | -------------
Screenshot 2020-10-16 at 16 59 35 | Screenshot 2020-10-16 at 16 59 05 

**How to test:** 
1. Log in as Admin and go the Item tab
2. Create an Item and go to Authoring
3. Drag and Drop Choice interaction to the white workspace
4. On the interaction, click "Response" and click on "Add an outcome"
5. Give a long identifier (as ex. Test_for_Outcome_Declarations)

### CODE REVIEW

File: views/scss/qtiCreator/_outcome-editor.scss [hunk 1]
```diff
@@ -29,6 +29,7 @@
             display: inline-block;
             margin: 0;
             width: calc(100% - 44px);
+            overflow: hidden;
```
Comment (jsconan) [inline views/scss/qtiCreator/_outcome-editor.scss:32]: **thought:** This looks a little rough. But it does the job.
**suggestion:** Could you add an ellipsis to ease the display for the user?

**Review by jsconan — COMMENTED** [review]
Code review only.

File: views/scss/qtiCreator/_outcome-editor.scss (hunk 1 above)
Comment (andrey-ponamarev) [reply views/scss/qtiCreator/_outcome-editor.scss:32]: yeah with ellipsis it looks much better

**Review by jsconan — APPROVED** [review]
**Review Checklist**
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

**Review by alexeyvy — APPROVED** [review]
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Pull request merged**